buildkite/src/Jobs/Release/MinaArtifactBullseyeDevnet.dhall

```diff
@@ -0,0 +1,28 @@
+let ArtifactPipelines = ../../Command/MinaArtifact.dhall
```

Comment:
Before we had a single file for both mainnet & devnet. It is impossible to calculate step key and step label based on parameters in a previous form. That's why i splitted mainnet & devnet and created separate files for each. A little more code but more control